superset/views.py

```diff
@@ -5,6 +5,7 @@
 
 from datetime import datetime, timedelta
 import json
+import simplejson as sjson
```

Comment:
I was avoiding the conflict as I moved the line `import simplejson as json` line from another file but can simply `import simplejson`